remixer_core/src/main/java/com/google/android/libraries/remixer/serialization/StoredVariable.java

```diff
@@ -122,12 +201,13 @@ public int hashCode() {
    */
   static StoredVariable fromRemixerItem(RemixerItem item) {
     StoredVariable storedVariable = null;
-    for (SupportedDataType type : SupportedDataType.values()) {
+    for (DataType type : Remixer.getInstance().getRegisteredDataType()) {
       try {
-        storedVariable = type.getValueConverter().fromRemixerItem(item);
+        storedVariable = type.getConverter().fromRemixerItem(item);
         break;
       } catch (IllegalArgumentException ex) {
         // Don't do anything, this just wasn't the right data type.
+        storedVariable = null;
```

Comment:
pmd complains for not having anything in a catch, not doing anything is the right thing in this case though.